<launch>
	
	<arg name="propulsion_address" default="128"/>

	<arg name="serial_port" default="/dev/nautonomous/actuation"/>
	<arg name="serial_available" default="false"/>

	<node name="nautonomous_actuation_propulsion_sabertooth" pkg="nautonomous_actuation_propulsion_sabertooth" type="sabertooth_propulsion_node" output="screen">
		<!-- Parameters -->
		<param name="serial_available" type="bool" value="$(arg serial_available)"/>
		<param name="propulsion_address" type="int" value="$(arg propulsion_address)"/>

		<param name="serial_port" value="$(arg serial_port)"/>

	</node>
	
</launch>
